### DISCUSSION TITLE
custom root folder

### DISCUSSION THREAD

**Discussion by mcosta74** [mkdocs/mkdocs#3044] [Q&A]
Hello, when you run `mkdocs init` the documentation is supposed to go in `docs` folder and it's not possible to load files outside that folder.
I'd like to have some .md files beside the source code; is it possible to specify that the root faolder for documentation is the project root (or a directory other that `docs`)?

**Comment by oprypin** [comment]
> directory other that docs

Sure, just change `docs_dir` config

> root faolder for documentation is the project root 

https://github.com/oprypin/mkdocs-same-dir

**Reply by mcosta74** [reply]
unfortunately looks like docs_dir can't be the root directory (it can't be a parent directory of the config file)
```
ERROR    -  Config value 'docs_dir': The 'docs_dir' should not be the parent directory of the config file. Use a child directory instead so that the
            'docs_dir' is a sibling of the config file.
```

**Reply by Andre601** [reply]
Why not do something like copying the files into the folder before build?
